src/tests/dictionaryConcepts/container/SelectAnswers.test.js

```diff
@@ -72,4 +88,51 @@ describe('<SelectAnswers />', () => {
     wrapper.find('#searchInputCiel').simulate('change');
     expect(spy).toHaveBeenCalled();
   });
+
+  it('should call resetInput  when a user types in a prepopulated answer to edit it', () => {
+    const newProps = {
+      ...props,
+      isShown: true,
+      isClicked: false,
+      answer: {
+        prePopulated: true,
+      },
+    };
+    wrapper = mount(<SelectAnswers {...newProps} />);
+    const spy = jest.spyOn(wrapper.instance(), 'resetInput');
+    wrapper.find('#searchInputCiel').simulate('keyDown');
+    expect(spy).toHaveBeenCalled();
+    expect(wrapper.state().isClicked).toBeTruthy();
+  });
+
+  it('should clear input and update with current value when a user presses any key', () => {
+    const newProps = {
+      ...props,
+      isShown: true,
+      isClicked: false,
+      answer: {
+        prePopulated: true,
+      },
+    };
+    wrapper = mount(<SelectAnswers {...newProps} />);
+    const event = { key: 'a' };
+    wrapper.find('#searchInputCiel').simulate('keyDown', event);
+    expect(setTimeout).toHaveBeenCalled();
```

Comment:
How is the above asserting that should clear input and update with current value when a user presses any key?

Reply:
Updated